docs/api.md

```diff
@@ -67,9 +67,10 @@ These are the API group versions which are curently available when using Antrea.
 
 ### Previously-supported CRDs
 
-| CRD | CRD version | Introduced in | Deprecated in | Removed in |
-|---|---|---|---|---|
+| CRD            | CRD version | Introduced in | Deprecated in | Removed in |
+|----------------|---|---|---|---|
 | `ClusterGroup` | v1alpha2 | v1.0.0 | v1.1.0 | v1.12.0 [^1] |
+| `Egress`       | v1alpha2 | v1.0.0 | v1.13.0 | v2.0.0 |
```

Comment:
If we follow the K8s deprecation policy, we may not be able to remove it in v2.0.0 considering Egress feature gate is already beta, @antoninbas @tnqn any comment?